docs/documentation/actions/groovy.md

```diff
@@ -0,0 +1,46 @@
+This action executes a [Groovy](https://groovy-lang.org/documentation.html){:target="_blank"} script.
```

Comment:
It could be worth mentionning that, as last statement of a Groovy script is the returned value and Chutney expects a map as output, the script must return a map either implicitly or explicitly, i.e. with or without return keyword.
So, for a script that just launch background execution for example, an empty map ([:]) must be the last statement.